### ISSUE TITLE
Pince from AUR crashes on start

### ISSUE THREAD

**Issue by terencode** [korcankaraokcu/PINCE#32]
```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
Detached from the process with PID:-1
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pexpect/spawnbase.py", line 150, in read_nonblocking
    s = os.read(self.child_fd, size)
OSError: [Errno 5] Input/output error

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pexpect/expect.py", line 99, in expect_loop
    incoming = spawn.read_nonblocking(spawn.maxread, timeout)
  File "/usr/lib/python3.6/site-packages/pexpect/pty_spawn.py", line 465, in read_nonblocking
    return super(spawn, self).read_nonblocking(size)
  File "/usr/lib/python3.6/site-packages/pexpect/spawnbase.py", line 155, in read_nonblocking
    raise EOF('End Of File (EOF). Exception style platform.')
pexpect.exceptions.EOF: End Of File (EOF). Exception style platform.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "PINCE.py", line 4309, in <module>
    window = MainForm()
  File "PINCE.py", line 300, in __init__
    GDB_Engine.init_gdb(gdb_path)
  File "/usr/share/PINCE/libPINCE/GDB_Engine.py", line 434, in init_gdb
    child.expect_exact("(gdb)")
  File "/usr/lib/python3.6/site-packages/pexpect/spawnbase.py", line 390, in expect_exact
    return exp.expect_loop(timeout)
  File "/usr/lib/python3.6/site-packages/pexpect/expect.py", line 105, in expect_loop
    return self.eof(e)
  File "/usr/lib/python3.6/site-packages/pexpect/expect.py", line 50, in eof
    raise EOF(msg)
pexpect.exceptions.EOF: End Of File (EOF). Exception style platform.
<pexpect.pty_spawn.spawn object at 0x7f3637343a58>
command: /usr/bin/sudo
args: [b'/usr/bin/sudo', b'LC_NUMERIC=C', b'./gdb_pince/gdb-8.0/bin/gdb', b'--interpreter=mi']
buffer (last 100 chars): ''
before (last 100 chars): 'sudo: ./gdb_pince/gdb-8.0/bin/gdb: command not found\r\n'
after: <class 'pexpect.exceptions.EOF'>
match: None
match_index: None
exitstatus: 1
flag_eof: True
pid: 22953
child_fd: 9
closed: False
timeout: None
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_string:
    0: "(gdb)"
```

**Comment by TsarFox** [comment]
Thank you for your bug report, this was a mistake on my part of hardcoding the GDB version into a regex used by the PKGBUILD.

Should be fixed now, can you confirm?

**Comment by terencode** [comment]
@TsarFox It works now, thank you :+1: 
Why would you hardcode that though?

**Comment by TsarFox** [comment]
@terencode Glad to hear it, and no problem.

Well, since you asked, the way PINCE is typically installed involves downloading the GDB source tree and compiling it - which for a very long time was the 7.11 stable release. There's one definition in PINCE for the path of the GDB binary, which if you clone from git is going to be pointing at where the install script puts GDB. When I was writing the PKGBUILD, I realized that it makes more sense to use the version of GDB from the Arch repos, so I added a line to the build that uses `sed` to change that definition to `/usr/bin/gdb`. The issue is that, because I never really expected the project to move onto a newer version of GDB, the regex for finding that definition contained the versioning information. In short, a lack of foresight on my part.
